packages/text-annotator-react/test/App.tsx

```diff
@@ -62,10 +62,11 @@ const MockStorage = () => {
 }
 
 export const App = () => {
-  
+  const w3cAdapter = useCallback((container: HTMLElement) => W3CTextFormat('https://www.gutenberg.org', container), []);
+
   return (
     <Annotorious>
-      <TextAnnotator>
+      <TextAnnotator adapter={w3cAdapter}>
```

Comment:
Simple one-liner definition for the formatter on the consumer side, where the `container` is always defined